### PR TITLE
backports: for v1.3.0

### PR DESCRIPTION
**Pull request by smira**
PRs backported:

- #6604 
- #6600
- #6628 
- #6615 

+ update Linux to 5.15.83

### CODE REVIEW

File: pkg/cluster/kubernetes/talos_managed.go [hunk 1]
```diff
@@ -126,6 +126,20 @@ func UpgradeTalosManaged(ctx context.Context, cluster UpgradeProvider, options U
 
 	options.Log("discovered controlplane nodes %q", options.controlPlaneNodes)
 
+	talosClient, err := cluster.Client()
```
Comment (frezbo) [inline pkg/cluster/kubernetes/talos_managed.go:129]: was this intended? the PR description misses this?

**Review by frezbo — APPROVED** [review]

File: pkg/cluster/kubernetes/talos_managed.go (hunk 1 above)
Comment (smira) [reply pkg/cluster/kubernetes/talos_managed.go:129]: it was a typo in the PR list, yes, I intended to backport the new checks you implemented!

**Pull request merged**